src/system-health/health_checker/hardware_checker.py

```diff
@@ -0,0 +1,247 @@
+from natsort import natsorted
+from swsssdk import SonicV2Connector
+from .health_checker import HealthChecker

```

Comment:
Add blank line to separate third-party from local imports:

```suggestion
from natsort import natsorted
from swsssdk import SonicV2Connector

from .health_checker import HealthChecker
```